data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -16,6 +16,10 @@ if ! release=$( podman inspect {{.ReleaseImage}} -f '{{"{{"}} index .RepoDigests
 	release="{{.ReleaseImage}}"
 fi
 
+if host -t SRV _etcd-server-ssl._tcp.{{.ClusterDomain}} | grep -q etcd-bootstrap; then
```

Comment:
optional improvement

if [[ $(host -t SRV _etcd-server-ssl._tcp.{{.ClusterDomain}} | grep -q etcd-bootstrap) ]]; then

Reply:
I believe it should be `grep bootstrap`, as `etcd-bootstrap` does not resolve

Reply:
> I believe it should be `grep bootstrap`, as `etcd-bootstrap` does not resolve

@hexfusion asked that I change the A record to be etcd-bootstrap instead of just bootstrap, so it is 100% clear what it is for.

Reply:
my bad, there is no existing A record. Agree that creating as `etcd-bootstrap` is a good idea